### PR TITLE
Create week 1 schedule.md

### PR DESCRIPTION
**Pull request by bobbysebolao**
Pretty much the same as the existing week 1 schedule, but with the new additions (like pre reading time) that we talked about.

Have replaced the day 1 GitHub scavenger hunt readme with a Kahoot quiz which went well with fac18.

### CODE REVIEW

File: week-1/schedule.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Teamwork & Toolkit Week Schedule
+
+## Day 1
+
+| Time    | Activity                                                                                                                                                                                                                                                                                                                                | Learning outcomes |
+| ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| 09:45   |                                                                                                                                                                                                                                                                                                                                         |                   |
```
Comment (oliverjam) [inline week-1/schedule.md]: Could we add 'Intro reading' here?

File: week-1/schedule.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+| 15:00   | [Technical spikes](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/week-1/research-afternoon.md)                                      |                                   |
+| 16:30   | Spike presentation prep                                                                                                                                            |                                   |
+| 17:00   | Spike presentations                                                                                                                                                |                                   |
+
+## Day 3
+
+| Time    | Activity                                                                     | Learning outcomes |
+| ------- | ---------------------------------------------------------------------------- | ----------------- |
+| 09:45   | Intro reading                                                                | CSS layouts       |
+| 10:00   | [CSS Gallery MC](https://github.com/foundersandcoders/css-gallery-challenge) | CSS Media queries |
```
Comment (oliverjam) [inline week-1/schedule.md]: Happy to leave this there for now, but I'm not sure it's a great intro to media queries. I think an image gallery is better suited to the CSS grid design burst stuff .

**Comment by oliverjam** [comment]
I'm not sure where to put it, but I'd really like to get my [Intro to HTML forms](https://github.com/oliverjam/learn-html-forms/) into Week 1.

I think it's really important to get a sense of how HTTP, requests and the web in general work before jumping into Week 2 (making AJAX requests, which is like a level up from basic forms).

File: week-1/schedule.md (hunk 1 above)
Comment (bobbysebolao) [reply week-1/schedule.md]: I think on day 1 of week 1 it'd be nice if people can spend this time chatting and getting to know each other.

**Comment by bobbysebolao** [comment]
@oliverjam ok, in that case let's swap the css gallery morning challenge for your learn HTML workshop and set pre-reading that relates to semantic HTML. Is there time for the CSS gallery challenge the following week instead?

File: week-1/schedule.md (hunk 1 above)
Comment (oliverjam) [reply week-1/schedule.md]: Ah yeah that's a better idea

**Comment by oliverjam** [comment]
@bobbysebolao cool. I'm not sure about that gallery workshop at all tbh. The relevant learning outcomes for Week 1 are currently "responsive design &  media queries" and "CSS grid". I think it would make sense to cover those in the design burst workshop rather than have this separate one.

File: week-1/schedule.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+| 14:00   | [Intro to Pair Programming](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/week-1/pair-programming.md)                                                                                                                                                                                                    | Pair programming  |
+| 15:00   | [Accessibility workshop](https://github.com/foundersandcoders/web-accessibility/blob/master/putting-yourself-in-someone-elses-shoes.md)                                                                                                                                                                                                 | Accessibility     |
+| 16:00   | GitHub scavenger hunt + [Kahoots quiz](https://create.kahoot.it/share/github-scavenger-hunt/f3992f24-666f-40cb-9ac5-56ede85e202d)                                                                                                                                                                                                       |                   |
```
Comment (oliverjam) [inline week-1/schedule.md]: Feedback from curriculum circle 07/02: swap this for estimation/user story material

**Comment by sofer** [comment]
@bobbysebolao thank you for adding the latest changes. 

Introducing all of the concepts in the estimation slides might be too much for Week 1. I think you took notes when we discussed this a couple of weeks ago, but my recollection is that we would attempt to build this up over the first 4 weeks. From memory, it was something like this:
1. User stories (set by us)
2. User stories with estimates (set by us)
3. User stories (set by us) with estimates (set by students)
4. Users stories and estimates (both set by students) 

That would mean that we would intro the following ideas in week 1:
- Scrum™
- Sprint
- User story
- Backlog
- Project board (Kanban)
- Sprint planning, retrospective and review (simple definitions)

And the following in Week 2:
- Estimate
- Sprint backlog
- Velocity
- Sprint planning and review (velocity-based  definitions)

Or something like that.
 
@oliverjam @Albadylic anything to add?

**Comment by sofer** [comment]
Have provisionally split the slide deck into two parts:
- https://hackmd.io/@sofer/S1wGfV2M8#/ (week 1)
- https://hackmd.io/@sofer/B1AL4V3ML#/ (week 2)

**Comment by sofer** [comment]
@oliverjam Have we got space for the second half of the issue management talk on Tuesday afternoon as part of the project intro (perhaps an extra 15 minutes would be enough if the project intro is kept short)?
- https://hackmd.io/@sofer/B1AL4V3ML#/

**Comment by oliverjam** [comment]
That sounds good. I'll have to find somewhere to squeeze that into Week 2

**Comment by bobbysebolao** [comment]
Great points all, I've updated the schedule to link to the project management slides @sofer , and have removed references to estimation/issue management in the project.md doc.

@sofer are you happy to merge?

**Comment by Albadylic** [comment]
This all looks good to me - the only thing is whether we have a space for Alex's talk on Impostor Syndrome mentioned at SGC. The first day's talks are all quite important, in my opinion. My suggestion would be for her to give the Wednesday talk on this topic.

File: week-1/schedule.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+| 14:00   | [Intro to Pair Programming](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/week-1/pair-programming.md)                                                                                                                                                                                                    | Pair programming  |
+| 15:00   | [Accessibility workshop](https://github.com/foundersandcoders/web-accessibility/blob/master/putting-yourself-in-someone-elses-shoes.md)                                                                                                                                                                                                 | Accessibility     |
+| 16:00   | [Intro to project management presentation(https://hackmd.io/@sofer/S1wGfV2M8#/)                                                                                                                                                                                                                                                         |                   | Project management |
```
Comment (oliverjam) [inline week-1/schedule.md]: You've got a missing bracket

```suggestion
| 16:00   | [Intro to project management presentation](https://hackmd.io/@sofer/S1wGfV2M8#/)                                                                                                                                                                                                                                                         |                   | Project management |
```

**Pull request merged**